code/modules/admin/topic.dm

```diff
@@ -359,7 +362,7 @@
 			return
 
 		message_admins("<span class='adminnotice'>[key_name_admin(usr)] is considering ending the round.</span>")
-		if(alert(usr, "This will end the round, are you SURE you want to do this?", "Confirmation", "Yes", "No") == "Yes")
+		if(alert(usr, "This will end the round, are you SURE you want to do this? [GLOB.admin_delay_notice ? "An admin has already delayed the round end for the following reason: [GLOB.admin_delay_notice]" : ""]", "Confirmation", "Yes", "No") == "Yes")

```

Comment:
Ending the round doesn't work when the round is delayed. You will have to do this in reboot-world.